site/content/integrate/apps/quickstart/_index.md

```diff
@@ -1,18 +1,63 @@
+
+Because they exist on a pre-defined network, other `docker-compose.yml` configurations can connect their containers to this development environment by specifying the same `mattermost-apps-dev` Docker network. Then the Mattermost server can be accessed at `http://mattermost:8065` by any other container on the `mattermost-apps-dev` network. Similarly, your own development apps can then be accessed by Mattermost via their service name (e.g., `http://mattermost-apps-typescript-hello-world:4000`). You can learn more about Docker networks on the [official documentation](https://docs.docker.com/network/).
+
+To change the Mattermost Apps plugin or the Mattermost server versions, you can edit [`docker-compose.yml`](https://github.com/mattermost/mattermost-app-examples/blob/master/docker-compose.yml).
+
+Next, use the following command to bring your development environment online.
+
+```sh
+docker compose up -d
+```
+
+Once Mattermost is online, you can access the server at [http://localhost:8065](http://localhost:8065) and create your superuser account. You can also add `127.0.0.1 mattermost` to your `/etc/hosts` file to be able to access Mattermost at [http://mattermost:8065](http://mattermost:8065), which is the value of `MM_SERVICESETTINGS_SITEURL` for Docker networking.
+
+To temporarily stop the container (and preserve your database), use the following command:
+
+```sh
+docker compose stop
+```
+
+To stop and clean up the database (for a fresh start), use the following command:
+
+```sh
+docker compose down
+```
+
+Now you're fully equipped with the ideal environment to develop Mattermost apps locally! 
+
+## Quick start guides
+
+The following quick start guides can help you start developing a Mattermost app in different languages:
 
 | Language                | Guide                                                                                             |
 |-------------------------|---------------------------------------------------------------------------------------------------|
-| Golang                  | [Quick start guide (Go)]({{< ref "/integrate/apps/quickstart/quick-start-go" >}})                 |
-| Golang (Serverless)     | [Quick start guide (serverless)]({{< ref "/integrate/apps/quickstart/quick-start-serverless" >}}) |
-| JavaScript / TypeScript | [Quick start guide (NodeJS)]({{< ref "/integrate/apps/quickstart/quick-start-js" >}})             |
-
-## Tests
+| Golang                  | [Hello World in Go]({{< ref "/integrate/apps/quickstart/quick-start-go" >}})                      |
+| Golang                  | [JWT in Go]({{< ref "/integrate/apps/quickstart/quick-start-jwt" >}})                                   |
```

Comment:
My thinking is we don't really have another place to put these kinds of guides right now. They're all very introductory looks at how Apps work, and follow the same kind of format and structure. They're also housed in the same repository. I think it makes sense to keep them together, until maybe we have more and it becomes more clear how to divide and reorganize them — and that would be a great problem to have! 🙂

I'd also eventually like to get the guides for lifecycle, oauth2, and webhooks set up in the same way (in future PRs). At that point, it would maybe make more sense to divide them into their own set of guides.